src/stamps.ts

```diff
@@ -153,17 +110,20 @@ export async function buyNewStamp(
   depth: number,
   amount: string,
   beeDebug: BeeDebug,
-  options: Options = {},
-): Promise<DebugPostageBatch> {
-  const batchId = await beeDebug.createPostageBatch(amount, depth)
+  options?: Options,
+): Promise<BatchId> {
+  const batchId = await beeDebug.createPostageBatch(amount, depth, {
+    waitForUsable: true,
+    waitForUsableTimeout: options?.timeout ?? DEFAULT_STAMP_USABLE_TIMEOUT,
```

Comment:
```suggestion
    waitForUsableTimeout: options?.timeout,
```
I would suggest using the bee-js's defaults instead.